collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -75,15 +75,11 @@
     public static final String TMPDRAWFILE_PATH = CACHE_PATH + File.separator + "tmpDraw.jpg";
     public static final String LOG_PATH = ODK_ROOT + File.separator + "log";
     public static final String DEFAULT_FONTSIZE = "21";
+    public static final int DEFAULT_FONTSIZE_INT = 21;
     public static final String OFFLINE_LAYERS = ODK_ROOT + File.separator + "layers";
     public static final String SETTINGS = ODK_ROOT + File.separator + "settings";
     private static Collect singleton = null;
 
-    static {
-        PRNGFixes.apply();
-        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

Comment:
@shobhitagarwal1612 Looks like this was something you added. Do you remember there being any reason to put it in the `static` block rather than in `onCreate`?

Reply:
@lognaturel I remember that when I tried to run the project for the first time, I was having `multi-dex` issues. So, I found this solution for that problem at a StackOverflow thread and I used it as it was mentioned. I think it would be better to move this code to `onCreate()` instead.